docs/source/collect/run.md

```diff
@@ -60,58 +88,41 @@ spec:
         imagePullPolicy: IfNotPresent
 
 ```
-## Examples using `imagePullSecret`
+## Examples using private images with `imagePullSecret`
 
-### Using dockerconfigjson secrets
+### Using a preexisting secret
```

Comment:
is this correct or should it be `pre-existing`? I'm not sure what the proper use is.

Reply:
I think actually both are accepted, oxford dictionary uses pre-exist and some american english dictionaries use preexist. I used it without hyphen but mark suggested otherwise, and I think it looks better with the hyphen...more readable maybe.

Reply:
I added the hyphen

Reply:
Cool, yeah that's what I found as well, agree the hyphen makes it more readable, thanks!